Added phpide:// links to line numbers

pmartin opened the pull request:
Line numbers in the left margin are now
hyperlinks to

```
phpide://_PROJECTROOT_/file@linenum
```

With `'file'` being the real path to the file
in the project, and linenum being the line number.

This will allow users to click on a line number and
have the corresponding file opened on the rigth line
in the IDE.
All they have to do is to configure their browser to
open `phpide://` links with a specific script, that will
extract file + line, and run their IDE, passing it
those parameters as it expects.

Note : the project's root is different on each developer's
computer -- and, so, it's their local script that has to
replace `'_PROJECTROOT_'` by the right actual project root.

This should, more or less, fix issue #10

Commit:
Added phpide:// links to line numbers

Line numbers in the left margin are now
hyperlinks to

    phpide://_PROJECTROOT_/file@linenum

With 'file' being the real path to the file
in the project, and linenum being the line number.

This will allow users to click on a line number and
have the corresponding file opened on the rigth line
in the IDE.
All they have to do is to configure their browser to
open phpide:// links with a specific script, that will
extract file + line, and run their IDE, passing it
those parameters as it expects.

Note : the project's root is different on each developer's
computer -- and, so, it's their local script that has to
replace '_PROJECTROOT_' by the right actual project root.

This should, more or less, fix issue #10

Commit:
Added calls to esc_specialchars() on files paths

jubianchi commented:
I would only add an ide:// scheme as Crew is able to handle non-PHP project (or I'm wrong ?)

KuiKui commented:
Can you also put your specific script in the 'scripts' directory ?

ratibus commented:
@jubianchi we are going to use crewide as prefix not phpide.

@KuiKui the script is specific to each developer. We are going to add a page in the wiki, explaining the configuration.

We'll add project name to the url, to be able to handle multiple projects.

Commit:
Fixes to [...]ide:// links

* Now using crewide:// protocol, instead of phpide://
  as crew is not php-only
* <a> tags in pure HTML, no more link_to (performance)
* The project's name (from crew) is now passed as a
  parameter to the script -- might help to figure out
  the absolute path to the file on the developer's
  computer

So, basically, the links now look like this :

    crewide://projectname@dir/dir/file.ext@123
